ui/packages/shared/profile/src/ProfileSelector/index.tsx

```diff
@@ -201,13 +208,21 @@ const ProfileSelector = ({
               error={error}
             />
           </div>
-          <div className="w-full flex-1">
+          <div className="flex w-full flex-1">
             <MatchersInput
               queryClient={queryClient}
               setMatchersString={setMatchersString}
               runQuery={setQueryExpression}
               currentQuery={query}
             />
+            <div>
+              <Input
+                className="divider-y block w-full flex-1 rounded rounded-l-none bg-gray-50 px-2 py-2 text-sm outline-none focus:ring-indigo-800 dark:bg-gray-900"
```

Comment:
The current styles don't exactly match the styles of the remainder of the top bar (spacing, rounding). I think we should have the same spacing around this as the other items in the top bar. To do so, we should probably move the Input to outside of the div that is holding the MatchersInput element.

Reply:
Unless we purposefully want it to look "attached" to the other filter input? In which case, we should modify that input to not be a text area. The bottom right corner of a textarea can be stretched/expanded. Once expanded the two filter inputs do not look related to one another.

Reply:
Yeah, exactly. It's supposed to be attached to the matchers query. It's basically an additional field for the query. I don't think we can move the textarea to be an input. Maybe the other way around would work though. Make the filter input a textarea?

Reply:
I guess given that it doesn't really match with the styles, I'll try to make them unattached for now.

Reply:
yeah I think unattached is the best way to go for now!